# Escribe un programa que intente dividir dos números. Si el segundo número es cero, captura la
# excepción ZeroDivisionError. Si el primer número es un número no válido, captura la excepción
# ValueError. En cualquier caso, muestra un mensaje de error al usuario.

print('\n==== 5. ZeroDivisionError / ValueError')

n1 = int(input('Ingresa 1er num (decimal para ValueError: '))
n2 = int(input('Ingresa 2do num(0 para ZeroDivisionError: '))
print(n1/n2)